source/index.js

```diff
@@ -36,15 +37,32 @@ var likely = function (node, options) {
 };
 
 /**
+ * Will be deprecated in 3.0
+ * @returns {Likely}
+ */
+var likely = function () {
+    // eslint-disable-next-line no-console
+    console.warn('This function will be DEPRECATED in 3.0. Use likely.initiate instead.');
```

Comment:
The warning is not entirely correct. This function is deprecated (i. e. marked for removal) _now_, and we’ll remove it in 3.0.